spec/features/locales_spec.rb

```diff
@@ -5,7 +5,7 @@
   }
     
   it 'changes to locale in Arabic when Arabic button is clicked' do
-    expect{ find("#change-to-arabic").click }.to change{ I18n.locale }.from(:en).to(:ar)
+    expect{ find("#change-to-arabic").click }.to change{ page.current_path }.from("/en").to("/ar")
     page_ok
```

Comment:
I spent a lot of time trying to use the `lang` attribute from the `html` tag without success. So, I implemented this solution instead.